apps/judicial-system/web/src/components/Conclusion/Conclusion.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+
+import { Box, Text } from '@island.is/island-ui/core'
+
+import BlueBox from '../BlueBox/BlueBox'
+import { conclusion as strings } from './Conclusion.strings'
+
+interface Props {
+  conclusionText?: string
+  judgeName?: string
+}
+
+const Conclusion: React.FC<Props> = (props) => {
+  const { formatMessage } = useIntl()
+  const { conclusionText, judgeName } = props
+
+  return !conclusionText ? null : (
```

Comment:
I feel like this would be more clear if turned around to have the affirmative action first, so conclusionText ? <BlueBox...> : null